### DISCUSSION TITLE
Daterime Picker

### DISCUSSION THREAD

**Discussion by MahmoudAlaaOmran** [TDRA-ae/aegov-dls#48] [General]
Hello,

Can you please add the date time picker element to the design system?

**Comment by moiz-conceptlz** [comment]
Hello @MahmoudAlaaOmran 

Just like the "Slider" and "Hero" components in the Design System recommend the use of Slick slider for jQuery and Embla.js for vanilla JavaScript, it is part of the roadmap to add a robust pre-existing date picker library that supports jQuery and Native JS.

However, as part of the upcoming ReactJS package of the design system, a Date Picker, along with a lot more components, will be added natively within the plugin. 

Stay tuned for this update. Follow our Roadmap on the designsystem.gov.ae website!
